js/main.js

```diff
@@ -21,5 +21,83 @@ function submitForm(event) {
   data.entries.unshift(formObject);
   $img.setAttribute('src', 'images/placeholder-image-square.jpg');
   $submit.reset();
+  document.getElementById('ul').prepend(renderEntry(formObject));
+  viewSwap('entries');
+  toggleNoEntries();
 }
 $submit.addEventListener('submit', submitForm);
+
+function renderEntry(entry) {
+  const $li = document.createElement('li');
+
+  const $divRow = document.createElement('div');
+  $divRow.setAttribute('class', 'row');
+  $li.appendChild($divRow);
+
+  const $divCol = document.createElement('div');
+  $divCol.setAttribute('class', 'column-full column-half');
+  $divRow.appendChild($divCol);
+
+  const $imgView = document.createElement('img');
+  $imgView.setAttribute('class', 'img-view-entry');
+  $imgView.setAttribute('src', entry.photoURL);
+  $divCol.appendChild($imgView);
+
+  const $divColLower = document.createElement('div');
+  $divColLower.setAttribute('class', 'column-full column-half');
+  $divRow.appendChild($divColLower);
+
+  const $h2 = document.createElement('h2');
+  $h2.textContent = entry.title;
+  $divColLower.appendChild($h2);
+
+  const $p = document.createElement('p');
+  $p.textContent = entry.notes;
+  $divColLower.appendChild($p);
+
+  return $li;
+}
+
+function appendEntry() {
+  for (let i = 0; i < data.entries.length; i++) {
+    document.getElementById('ul').appendChild(renderEntry(data.entries[i]));
```

Comment:
Once you fix the comment above, you can just reuse the same variable name here for the `ul`